doc/modules/changes.h

```diff
@@ -7,6 +7,11 @@
  *
  * <ol>
  *
+ * <li> The manual.pdf is no longer part of the git repository but you can
+ * find it online at aspect.dealii.org or you can build it yourself.
```

Comment:
Please provide a full link with `http://` so that doxygen automatically produces the link. Ok with that change.